fdbclient/SystemData.h

```diff
@@ -223,6 +223,13 @@ extern const KeyRef excludedServersVersionKey; // The value of this key shall be
 const AddressExclusion decodeExcludedServersKey(KeyRef const& key); // where key.startsWith(excludedServersPrefix)
 std::string encodeExcludedServersKey(AddressExclusion const&);
 
+extern const KeyRef excludedLocalityPrefix;
+extern const KeyRangeRef excludedLocalityKeys;
+extern const KeyRef excludedLocalityVersionKey; // The value of this key shall be changed by any transaction that
+                                               // modifies the excluded localities list
+const std::string decodeExcludedLocalityKey(KeyRef const& key); // where key.startsWith(excludedLocalityPrefix)
```

Comment:
Functions should ~never return a const value. It provides a small amount of protection from trying to mutate a temporary (e.g. `decodeExcludedLocalityKey(...) = "foo"` is now a compilation error), but it pessimizes standard practices (e.g. `foo = decodeExcludedLocalityKey(...);` is now a copy instead of a move) and provides limited semantic benefit since a client can simply copy the value and mutate away.